libafl/src/stages/calibrate.rs

```diff
@@ -365,6 +367,29 @@ where
             track_stability: false,
             restart_helper: ExecutionCountRestartHelper::default(),
             phantom: PhantomData,
+            name: Cow::Borrowed("CalibrationStage"),
         }
     }
+    /// Create a new [`CalibrationStage`], naming it as expected by AFL++.
+    #[must_use]
+    pub fn with_afl_name<F>(map_feedback: &F) -> Self
+    where
+        F: HasObserverHandle<Observer = C> + Named,
+    {
+        Self {
+            map_observer_handle: map_feedback.observer_handle().clone(),
+            map_name: map_feedback.name().clone(),
+            stage_max: CAL_STAGE_START,
+            track_stability: true,
+            restart_helper: ExecutionCountRestartHelper::default(),
+            phantom: PhantomData,
+            name: Cow::Borrowed("calibration"),
```

Comment:
I would make this a constant anyway, like `AFL_NAME` - then we can also reference it from other parts of the code